src/clients/migration_proposals.rb

```diff
@@ -1,3 +1,23 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) 2015 SUSE GmbH, All Rights Reserved.
```

Comment:
hmm, I use it same as in spec file which is done automatically so I expect it have the newest copyright holder as it also refresh date: https://build.suse.de/package/view_file/SUSE:SLE-12:GA/MozillaFirefox/MozillaFirefox.spec?expand=1